CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# Contributing to React Native CLI
+
+## Development Process
+
+All work on React Native CLI happens directly on GitHub. Contributors send pull requests which go through review process.
+
+> **Working on your first pull request?** You can learn how from this *free* series: [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github).
+
+1. Fork the repo and create your branch from `master` (a guide on [how to fork a repository](https://help.github.com/articles/fork-a-repo/)).
+2. Run `yarn` or `npm install` to install all required dependencies.
+3. Now you are ready to do the changes.
+
+## Repository
+
+Repository is splitted into two packages:
+
+* `local-cli` - Historically, it was included in `react-native` package. Contains all the commands code.
+* `global-cli` - Historically, it was a `react-native-cli` package and the only reason this package existed was to initialize an empty project.
+
+## Testing your changes
+
+To test your changes you will need to have another `react-native` project. You can initialize new one or use already existing repository. Next, you will need to install `local-cli` with your changes:
+```sh
+# Assuming that you have react-native-cli cloned in the parent directory
+yarn add ../react-native-cli/packages/local-cli
+```
```

Comment:
Would suggest updating this to have two ways:

1) Using "link" (haven't tested myself)
2) Calling "cli.js" file like this: "node ../<path_to_your_rn_cli>/cli.js". The project root will be automatically recognised. Make sure the version of React Native matches the one present in devDependencies of the CLI. Otherwise, you may get unexpected errors.